tests/internal/CMakeLists.txt

```diff
@@ -266,9 +266,9 @@ if(ENABLE_CLANGQUERY)
 endif()
 
 #------------------------------------------------------------------------------
-# Format the testing code using AStyle
+# Format the testing code using ClangFormat
```

Comment:
Since this PR doesn't currently run ``clang-format`` and doesn't have a ``.clang-format`` file, should we revert this file back to the ``astyle`` version?

Reply:
well it technically does have a .clang-format file.. i was using the one included in google benchmark. But i will split the PR into two.